bindings/python/iota_sdk/wallet/wallet.py

```diff
@@ -805,6 +805,29 @@ def send_outputs(
             }
         ))
 
+    def send_mana(
+            self, params: SendManaParams, options: Optional[TransactionOptions] = None) -> TransactionWithMetadata:
+        """Send mana.
+        """
+        return TransactionWithMetadata.from_dict(self._call_method(
```

Comment:
should call `prepare_send_mana` like the other send bindings

Reply:
Not 100% sure about this since it requires two calls to the Rust side, but if we want it everywhere we also don't need the direct methods like `SendMana` in the bindings core

Reply:
Ah we commented at the same time. But that's what we're doing everywhere already

Reply:
But yeah we should probably do it everywhere to have less "duplication"

Reply:
They're also in the high level mod :trollface:
`SendOutputs` is the only not high level one I would say, but even high level or not, why have it different for them?

Reply:
what's the plan?

Reply:
Removing `SendMana` from the bindings core is the plan https://github.com/iotaledger/iota-sdk/pull/2068#discussion_r1504854601 :trollface:
For the others I opened an issue https://github.com/iotaledger/iota-sdk/issues/2081